### PR TITLE
[14.0][hr_attendance_hour_type_report] update FR translation

### PR DESCRIPTION
**Pull request by ajaniszewska-dev**

### CODE REVIEW

File: hr_attendance_hour_type_report/i18n/fr.po [hunk 1]
```diff
@@ -42,12 +41,12 @@ msgstr ""
...
 #: model:ir.model,name:hr_attendance_hour_type_report.model_res_company
 msgid "Companies"
-msgstr ""
+msgstr "Sociétés"
 
 #. module: hr_attendance_hour_type_report
 #: model:ir.model,name:hr_attendance_hour_type_report.model_res_config_settings
 msgid "Config Settings"
-msgstr ""
+msgstr "Paramètres de config"
```
Comment (flotho) [inline hr_attendance_hour_type_report/i18n/fr.po:49]: Hi, thanks for your contribution.
could you use the whole word for `configuration`

File: hr_attendance_hour_type_report/i18n/fr.po [hunk 2]
```diff
@@ -161,10 +160,4 @@ msgstr ""
...
 #: model:ir.model.fields,help:hr_attendance_hour_type_report.field_res_config_settings__hr_night_work_hour_start
 msgid "expressed in company timezone"
-msgstr ""
-
-#~ msgid "Invoice section name"
-#~ msgstr "Nom de section sur facture"
-
-#~ msgid "Sales Order"
-#~ msgstr "Bon de commande"
+msgstr ""
```
Comment (flotho) [inline hr_attendance_hour_type_report/i18n/fr.po:163]: please add new line here to respect pylint

**Review by flotho — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
Please use Weblate for this:

https://odoo-community.org/page/translate

**Comment by flotho** [comment]
oh yes, I forgot this one, it will be updated by robot!

**Comment by flotho** [comment]
@ajaniszewska-dev  log to https://translation.odoo-community.org/translate/hr-attendance-14-0/hr-attendance-14-0-hr_attendance_hour_type_report/fr/?q=state:%3Ctranslated and ask OCA for access right

**Pull request closed without merging**